### ISSUE TITLE
SCOM 2016 uses new certificate store

### ISSUE THREAD

**Issue by Aaron-** [EUCweb/BIS-F#37]
SCOM 2016 is using a new certificate store named "Microsoft Monitoring agent" so the current SCOM prep script does not work. 
Working command for scom 2016:
`& Invoke-Expression "certutil -delstore ""Microsoft Monitoring Agent"" 0"`

**Comment by EUCweb** [comment]
thx, AAron. can you send me back the right version of SCOM 2016 or can send me back the current BIS-F preparation log, to see the version of SCOM is detected during sealing

**Comment by Aaron-** [comment]
Here you go
27-3-2018 23:10:08 | xxx$ | INFORMATION... | Product Microsoft SCOM Agent (Version 8.0.10918.0) installed
27-3-2018 23:10:08 | xxx$| INFORMATION... | Path C:\Program Files\Microsoft Monitoring Agent\Agent\Health Service State detected

**Comment by EUCweb** [comment]
Aaron, please take a copy of 10_PrepBISF_SCOM.ps1 from https://github.com/EUCweb/BIS-F/tree/bugfix/37_SCOM_CertStore to see if works

**Comment by Aaron-** [comment]
Works, first certutil command returns an error (as expected), the 2nd one will delete the certificate:
```
`PS C:\Program Files (x86)\Base Image Script Framework (BIS-F)\Framework\SubCall\Global> c:\...\test.ps1
+ Product Microsoft SCOM Agent installed
        > Reconfigure Service HealthService
Operations Manager
CertUtil: -delstore command FAILED: 0x80070002 (WIN32: 2 ERROR_FILE_NOT_FOUND)
CertUtil: Het systeem kan het opgegeven bestand niet vinden.
Microsoft Monitoring Agent
Deleting Certificate 0: O=Microsoft, OU=RunAs Account Encryption, CN=
```
`

**Comment by EUCweb** [comment]
pls got a second copy of the script, to see if the it works without the error output

**Comment by Aaron-** [comment]
Used the script from https://github.com/EUCweb/BIS-F/blob/a9d1603f0c406b75c5dbdb0bdc9f857627fd7ebc/Framework/SubCall/Preparation/10_PrepBISF_SCOM.ps1 with the erroraction silently continue, still gives the same error.

Would it be an option to use the following (supported powershell 3.0 onwards)
`get-childitem "Cert:\LocalMachine\Microsoft Monitoring Agent" -errorAction SilentlyContinue | %{ remove-item $_.PSpath }
get-childitem "Cert:\LocalMachine\Operations Manager" -errorAction SilentlyContinue | %{ remove-item $_.PSpath }`
```

**Comment by EUCweb** [comment]
updated it now wit a try / catch errorhandling in, please test it again https://github.com/EUCweb/BIS-F/blob/develop/Framework/SubCall/Preparation/10_PrepBISF_SCOM.ps1

**Comment by Aaron-** [comment]
Works without error :)

**Comment by vashan** [comment]
When i run the BIS-F Script the Microsoft Monitoring Agent Service "Healthservice is stopped and set to manual startup. I tried to create Powershell script to start the service and set to automatic start. I placed the script to the "Personalization\Custom" folder. But the script is not running on computer startup. I need the service to startup and set to automatic startup. How can i solve this. Why is the service set stopped and set to manual startup by the BIS-F Script?
